### PR TITLE
Objects Automated Test - LPS-141611 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-141611

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 1]
```diff
@@ -2934,12 +2934,47 @@ definition {
...
+		ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+			fieldName = "customObjectField",
+			objectName = "CustomObject",
+			value = "123.123456");
+
+		ObjectAdmin.goToCustomObject(objectName = "CustomObject");
 
+		ObjectPortlet.addSingleFieldEntryViaUI(
+			entry = "987.987654",
+			entryOption = "123.123456");
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:2977]: You don't need to add an entry for this test. You only need to click on the Relationship field and assert the entry value. You can use the `CreateObject.assertAddedEntryRelationshipIsDisplayed` macro for this. See example on the `CanViewObjectEntryTitleStringOnRelationshipField` test.

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro [hunk 2]
```diff
@@ -3,10 +3,18 @@ definition {
 	macro addSingleFieldEntryViaUI {
 		LexiconEntry.gotoAdd();
 
-		Type.type(
+		Type(
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro:6]: You don't need this, you can remove.

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro [hunk 3]
```diff
@@ -3,10 +3,18 @@ definition {
...
 			value1 = "${entry}");
 
+		if (isSet(entryOption)) {
+			Click(locator1 = "ObjectPortlet#SEARCH_RELATIONSHIP_ENTRY");
+
+			Click(
+				key_picklistOption = "${entryOption}",
+				locator1 = "ObjectAdmin#ENTRY_PICKLIST_OPTION");
+		}
+
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro:17]: You don't need this, you can remove.

**Review by rodrigocunhaa — CHANGES_REQUESTED** [review]
Hi @andreefarias ,

Please, check my comments and make the necessary changes.

Let me know if you have any question.

Thanks!

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase (hunk 1 above)
Comment (andreefarias) [reply modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase]: Reviewed! I thought other way. It was more simple.

Thanks!

**Comment by rodrigocunhaa** [comment]
ci:forward

**Pull request closed without merging**